source/release-notes.txt

```diff
@@ -10,6 +10,22 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.1
+----------------
+
+*Released August 9, 2023*
+
+New Features:
+
+* Show insight for unbound array (:issue:`COMPASS-6836`)
+* Use modal to highlight legacy connections (:issue:`COMPASS-7072`)
+
+Bug Fixes:
+
+* Split connection storage between processes (:issue:`COMPASS-7078`)
```

Comment:
Slightly unclear what this means for this Bug Fix, could you clarify with the Compass team on both bullet points to add a bit more detail?

Reply:
Fixed